_posts/2018-07-20-exploring-state-management-in-react-container-components.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title: Exploring State Management in React&#58; Container Components
+date: 2018-07-20 09:00:00 -04:00
+categories: post
+desc: An amazing blog post, truly one of the best! PS CHANGEME
+permalink: /:categories/:year/:month/:day/:title/
+---
+
+>When should we install Redux?
+
+This question is often asked, and answers still vary wildly. The truth is there is quite a bit you can do before needing to pull in Redux, and even then Redux isn't your only option! 
```

Comment:
`is often asked` is passive voice, which (while not evil) can be switched to active voice for greater impact: "People less familiar with state-based application development often ask this question" or similar. I'd use this to connect with my audience.

Reply:
I've never actually understood what `passive voice` meant. Thanks for driving me to actually look it up!

[Page I referenced](https://writing.wisc.edu/Handbook/CCS_activevoice.html)

I'm now realizing I've been writing in the passive voice for a long time.